.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,36 @@
+name: Golint
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**.go'
+      - '.github/workflows/golangci-lint.yml'
+  pull_request:
+    paths:
+      - '**.go'
+      - '.github/workflows/golangci-lint.yml'
+
+jobs:
+  golangci:
+    name: Format
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go_version: ['1.16', '1.18']
```

Comment:
1.17 instead ?

Reply:
oh I can add 1.17, in `go.mod` im using 1.18 which I think is the most recent stable version?

Reply:
Yep 1.18 should be the main target now.